src/translations/components/TranslationsProductsPage/TranslationsProductsPage.tsx

```diff
@@ -163,22 +164,25 @@ const TranslationsProductsPage: React.FC<TranslationsProductsPageProps> = ({
             initialState={true}
             title={intl.formatMessage(commonMessages.translationAttributes)}
             fields={
-              data.attributeValues.map((attrVal, i) => ({
+              data.attributeValues.map<TranslationField>(attrVal => ({
                 id: attrVal.attributeValue.id,
                 displayName: intl.formatMessage(
                   {
-                    id: "PajjqE",
-                    defaultMessage: "Attribute {number}",
+                    id: "zgqPGF",
+                    defaultMessage: "Attribute {name}",
                     description: "attribute list",
                   },
                   {
-                    number: i + 1,
+                    name: attrVal.attribute.name,
                   },
                 ),
-                name: attrVal?.name,
-                translation: attrVal?.translation?.richText || null,
-                type: "rich" as "rich",
-                value: attrVal?.richText,
+                name: attrVal.name,
+                translation:
+                  attrVal.translation?.richText ||
+                  attrVal.translation?.plainText ||
+                  null,
```

Comment:
so, you repeat the same logic, a function perhaps, and reuse it?

Reply:
Good idea, I added a function for it.